modules/test/playwright/tests/frontend-data-set-views-web/pages/view/visualization_modes/VisualizationModesPage.ts

```diff
@@ -76,6 +76,31 @@ export class VisualizationModesPage {
 			.waitFor();
 	}
 
+	async openChangeFieldModal({
+		container,
+		sectionLabel,
+	}: {
+		container: Locator;
+		sectionLabel: string;
+	}) {
+		await container
+			.locator('tr')
+			.filter({has: this.page.getByText(sectionLabel)})
+			.getByTitle(`View ${sectionLabel} Options`)
+			.click();
+
+		const changeAssignmentButton = this.page.getByRole('menuitem', {
+			name: 'Change Assignment',
+		});
+
+		await changeAssignmentButton.waitFor();
+		await changeAssignmentButton.click();
+
+		await this.fieldSelectModalContainer
+			.getByPlaceholder('Search')
+			.waitFor();
+	}
+
```

Comment:
@chestofwonders I copied this bit from https://github.com/liferay-frontend/liferay-portal/pull/4055. It is the part that is fixing tests that recently got broken by addition of list clearing.